pkg/config/config.go

```diff
@@ -46,6 +48,31 @@ type Config struct {
 	FedRAMP      bool     `json:"fedramp,omitempty" doc:"Indicates FedRAMP."`
 }
 
+var DisallowedSetConfigProperties = []string{"scopes"}
+
+func ConfigPropertiesNamesAndDocs() ([]string, []string) {
```

Comment:
@tirthct Could we add a unit test for this please

Reply:
Added tests